src/widgets/NftDetailWidget/NftDetailWidget.scss

```
@@ -35,6 +16,22 @@
         }
     }
 
+    &__main-info {
+        &--tablet-only {
+            display: none;
+
+            p {
+                margin-bottom: 0;
+            }
+        }
```

If you want to hide things on mobile and show on tablet do this:

```suggestion
     &__main-info {
        display: none;

        @include for-size(tablet-portrait-up) {
            display: block;
        }
    }
```

---

Poorly named modifier, but it's a separate behaviour to the default. It's easier to see with the current changes. My brain is tired, I'll have a ponder re how to update/replace this.

---

We have the `for-size` mixin for responsive styles. Let's not mix things up.